features/enable_fips_cloud.feature

```diff
@@ -466,3 +466,22 @@ Feature: FIPS enablement in cloud based machines
            | release | machine_type |
            | bionic  | aws.generic  |
            | focal   | aws.generic  |
+
+    Scenario Outline: Attached enable of FIPS in an ubuntu GCP vm
```

Comment:
I'm not worrying that much now as we never did in the past, but at some point in time we need to verify the Scenario names... even the ones that are correct could be more descriptive

Reply:
Indeed